go/tao/hosted_program_factory.go

```diff
@@ -35,5 +35,5 @@ type HostedProgram interface {
 // containing a new instance of an operating system.
 type HostedProgramFactory interface {
 	MakeSubprin(uint, string, int, int) (auth.SubPrin, string, error)
-	Launch(string, []string, int, int) (io.ReadWriteCloser, HostedProgram, error)
+	Launch(string, []string, int, int, []int) (io.ReadWriteCloser, HostedProgram, error)
```

Comment:
I think this is a reasonable change, since LinuxHost is always going to be starting something under a different process, whether it be a docker container, a qemu instance, or another process. However, I'm starting to find the parameters fairly opaque; it was easy to understand when it was string, string[], but now I think we need at least a comment and maybe actual parameter names.